src/main.rs

```diff
@@ -4,11 +4,8 @@
 
 #![no_std]
 #![no_main]
-#![feature(core_panic)]
 #![feature(naked_functions)]
-#![feature(str_split_whitespace_as_str)]
-#![forbid(clippy::inline_asm_x86_intel_syntax)]
-#![warn(rust_2018_idioms)]
+#![feature(str_split_whitespace_remainder)]
```

Comment:
Fancy also bumping the copyright message year in the message that gets printed on startup?

Reply:
Done.

I also updated the copyright year in the files that were changed (well, those where I could or it made sense to do so).